### ISSUE TITLE
hamsket 0.6.2 crashes on launch

### ISSUE THREAD

**Issue by aravindhp** [TheGoddessInari/hamsket#245]
Hamsket now crashes on launch:

```
 /opt/Hamsket/hamsket 
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'electron-is-dev'
Require stack:
- /opt/Hamsket/resources/app.asar/electron/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:961:15)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:257:921)
    at Module._load (internal/modules/cjs/loader.js:844:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/opt/Hamsket/resources/app.asar/electron/main.js:11:15)
    at Module._compile (internal/modules/cjs/loader.js:1145:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1166:10)
    at Module.load (internal/modules/cjs/loader.js:981:32)
```

### Steps to reproduce

1. Install Hamsket 0.6.2
2. Launch it
3. 

### Expected behavior

Hamsket should not crash

### Actual behavior

Hamsket crashes

### ENVIRONMENT

> Hamsket: v0.6.2

> OS: Fedora 33

> Arch: x64

> BuildVersion: 0.6.2

> Does this happen upstream: N/A

**Comment by wallentx** [comment]
Also same thing on 0.6.3 nightly

**Comment by Naticus** [comment]
Can confirm this also happens on Win10 v20H2 with Hamsket v0.6.2.

**Comment by vurtomatic** [comment]
Came to report the same. MacOS 10.15.7 with Hamsket 0.6.2.

**Comment by eaganlai** [comment]
Came to report the same. Win 10 Pro 120.2212.551.0 with Hamsket 0.6.2 and Nightly.

**Comment by niczak** [comment]
Same issue here, Windows 10 Pro v20H2 64-bit. 
![image](https://user-images.githubusercontent.com/341180/101860244-faf9a380-3b32-11eb-9416-c402319d4171.png)

**Comment by LiBOSS-CZ** [comment]
Same for me but not for 0.6.2 but 0.6.3 (i think only for 0.6.3 because it named 0.6.2 in the archive) - probadly downgrade of electron.
For me helped downgrade to 0.6.2 - 323503b

**Comment by aravindhp** [comment]
This issue has been fixed in the re-released [0.6.2](https://github.com/TheGoddessInari/hamsket/releases/tag/0.6.2)
